Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN ./google-cloud-sdk/install.sh --quiet
 
 # Install kubectl
 RUN ./google-cloud-sdk/bin/gcloud components install kubectl
+RUN ./google-cloud-sdk/bin/gcloud components update --quiet
```

Comment:
this overrides the version specified above, which could be surprising.

I'd generally be in favor of just using the latest version, but seem to remember there was a reason for pinning it (but don't remember what)

Reply:
ok, so pinning versions goes all the way back to the beginning, so I'm going to assume there is no good reason for it.

I'm also really confused as to the FROM image -- I could have sworn I saw `FROM google/cloud-sdk:161.0.0-alpine`, but master and develop are now both `FROM alpine:3.4`, and that also goes back to the beginning. It's possible I messed something up when I tried to make develop match master.

Anyway, is there any reason to not just use `FROM google/cloud-sdk:latest` and be done with all this? I think we've been through this process many times, and the risk is low.

Reply:
Ok with using `FROM google/cloud-sdk:latest` as the default setup.

@andrew-canaday if you can remove the update as well.

Reply:
Updated, as per your comments and pushed. Thanks!